lib/ts/recipe/session/types.ts

```diff
@@ -169,15 +174,25 @@ export interface TokenTheftErrorHandlerMiddleware {
     (sessionHandle: string, userId: string, request: BaseRequest, response: BaseResponse): Promise<void>;
 }
 
+export interface MissingClaimErrorHandlerMiddleware {
+    (validatorError: ClaimValidationError, request: BaseRequest, response: BaseResponse): Promise<void>;
+}
+
 export interface NormalisedErrorHandlers {
     onUnauthorised: ErrorHandlerMiddleware;
     onTryRefreshToken: ErrorHandlerMiddleware;
     onTokenTheftDetected: TokenTheftErrorHandlerMiddleware;
+    onInvalidClaim: MissingClaimErrorHandlerMiddleware;
 }
 
 export interface VerifySessionOptions {
     antiCsrfCheck?: boolean;
     sessionRequired?: boolean;
+    overwriteDefaultValidators?: (
```

Comment:
Hmm, I thought I already renamed it to global... Also, the override may be more fitting here, I just didn't want to use it because this is very different from the normal overrides we use.

Reply:
I think it's fine. We can come back to this when we discuss the interface once again with the team - keeping this comment open until then